src/main.rs

```diff
@@ -1,40 +1,8 @@
-use anyhow::Result;
+mod cli;
 
-use lurk::eval::lang::{Coproc, Lang};
-use lurk::field::LanguageField;
-use lurk::repl::{repl_cli, ReplState};
-use pasta_curves::{pallas, vesta};
+use anyhow::Result;
 
 fn main() -> Result<()> {
     pretty_env_logger::init();
-
-    let default_field = LanguageField::Pallas;
-    let field = if let Ok(lurk_field) = std::env::var("LURK_FIELD") {
-        match lurk_field.as_str() {
-            "BLS12-381" => LanguageField::BLS12_381,
-            "PALLAS" => LanguageField::Pallas,
-            "VESTA" => LanguageField::Vesta,
-            _ => default_field,
-        }
-    } else {
-        default_field
-    };
-
-    match field {
-        LanguageField::BLS12_381 => repl_cli::<
-            blstrs::Scalar,
-            ReplState<blstrs::Scalar, Coproc<blstrs::Scalar>>,
-            Coproc<blstrs::Scalar>,
-        >(Lang::<blstrs::Scalar, Coproc<blstrs::Scalar>>::new()),
-        LanguageField::Pallas => repl_cli::<
-            pallas::Scalar,
-            ReplState<pallas::Scalar, Coproc<pallas::Scalar>>,
-            Coproc<pallas::Scalar>,
-        >(Lang::<pallas::Scalar, Coproc<pallas::Scalar>>::new()),
-        LanguageField::Vesta => repl_cli::<
-            vesta::Scalar,
-            ReplState<vesta::Scalar, Coproc<vesta::Scalar>>,
-            Coproc<vesta::Scalar>,
-        >(Lang::<vesta::Scalar, Coproc<vesta::Scalar>>::new()),
-    }
+    cli::parse()
```

Comment:
This reads very strangely if one doesn't know that `cli::parse()` does much more than parse.